internal/ate/stacks_test.go

```diff
@@ -154,6 +155,41 @@ func TestHeaderStacks(t *testing.T) {
 				},
 			}},
 		},
+	}, {
+		desc: "ethernet header w/ ethernet-type",
```

Comment:
Can you just add an etherType field to the "ethernet header" case instead. In fact, if you could also do a bit of cleanup and get rid of the "ethernet header w/ vlan id" and do the same thing there (just add the vlan id to the header), that would be appreciated.

Reply:
sure. merged those 2 sub testcases as suggested. thanks